Source/Immutable/Private/Immutable/ImmutablePassport.cpp

```diff
@@ -433,16 +433,31 @@ void UImmutablePassport::OnLogoutResponse(FImtblJSResponse Response) {
   if (auto ResponseDelegate = GetResponseDelegate(Response)) {
     FString Msg;
     if (Response.success) {
+      FString Err;
+      // Todo: This is an interim solution until we have logout URL and redirect page 
+      // fully implemented by Passport team. https://immutable.atlassian.net/browse/ID-1238
+      FString LogoutUrl = "https://auth.immutable .com/v2/logout";
+      FPlatformProcess::LaunchURL(*LogoutUrl, nullptr, &Err);
```

Comment:
This logout function will be used regardless of whether the user logged in using the device auth or PKCE flow. Because of this, can we please ensure that the user experience is consistent for both login and logout?

Device flow
- Login: the external browser opens for the user to log in
- Log out: the external browser opens, which properly logs the user out

PKCE flow (Android and iOS)
- Login: an embedded browser is rendered on top of the game for the user to log in
- Log out: an embedded browser is rendered on top of the game, which properly logs the user out

PKCE flow (macOS)
- Login: a small window is shown for the user to log in
- Log out: a small window is shown which properly logs the user out

Reply:
As discussed with @nattb8,

The current logout implementation will work well for Device flow only. Might need seperate implementation for PKCE flow due to the login flow and UX is different.

The solution discussed is to implement a seperate logout methods for both PKCE and DeviceFlow inline with their corresponding login methods.

Reply:
👍 Let's add docs to this function that it is recommended to be used alongside the device flow login.

Reply:
Sure will add it along with https://immutable.atlassian.net/browse/DX-2518